lib/src/fix.rs

```diff
@@ -291,11 +322,29 @@ pub fn fix_files(
                         file_id: id.clone(),
                         repo_path: repo_path.clone(),
                     };
-                    if let Some(new_id) = fixed_file_ids.get(&file_to_fix) {
-                        return Some(TreeValue::File {
-                            id: new_id.clone(),
-                            executable: *executable,
-                        });
+                    if let Some(fix_result) = fixed_file_ids.get(&file_to_fix) {
+                        match fix_result {
+                            FixResult::Fixed(new_file_id) => {
+                                return Some(TreeValue::File {
+                                    id: new_file_id.clone(),
+                                    executable: *executable,
+                                });
+                            }
+                            FixResult::Unchanged => {
+                                // TODO: add counters for this in FixSummary
+                                todo!()
+                            }
+                            FixResult::InvalidContent(_info) => {
+                                // TODO: include this information in FixSummary, keyed by commit_id
+                                // and repo_path.
+                                todo!()
+                            }
+                            FixResult::FileTooLarge => {
+                                // TODO: include this information in FixSummary, keyed by commit_id
+                                // and repo_path.
+                                todo!()
+                            }
+                        }
```

Comment:
nit: I think its better to just add the `FixResult::Fixed` and `FixResult::Unchanged` cases for this PR and then do the rest one by one or in followups

Reply:
I like your suggestion. I will break it up, unless I get other feedback. I am going to be on vacation starting tomorrow coming back on April 13, so it is possible I won't get to it until I get back.

Reply:
> I like your suggestion.

Thanks. 

> I am going to be on vacation starting tomorrow coming back on April 13, so it is possible I won't get to it until I get back.

Take your time and enjoy your vacation.